packages/cli/generation/ir-generator/src/examples/ExampleGenerator.ts

```diff
@@ -0,0 +1,616 @@
+                return {
+                    jsonExample: `"${literal.string}"`,
+                    shape: ExampleTypeReferenceShape.primitive(ExamplePrimitive.string({ original: literal.string }))
+                };
+            default:
+                assertNever(literal);
+        }
+    }
+
+    private generateExamplePrimitive(primitiveType: PrimitiveType): ExampleTypeReference {
+        switch (primitiveType) {
+            case "STRING":
+                return {
+                    jsonExample: "string",
+                    shape: ExampleTypeReferenceShape.primitive(ExamplePrimitive.string({ original: "string" }))
+                };
+            case "INTEGER":
+                return {
+                    jsonExample: 42,
+                    shape: ExampleTypeReferenceShape.primitive(ExamplePrimitive.integer(0))
+                };
+            case "DOUBLE":
+                return {
+                    jsonExample: 1.0,
+                    shape: ExampleTypeReferenceShape.primitive(ExamplePrimitive.double(1.0))
+                };
+            case "BOOLEAN":
+                return {
+                    jsonExample: true,
+                    shape: ExampleTypeReferenceShape.primitive(ExamplePrimitive.boolean(true))
+                };
+            case "LONG":
+                return {
+                    jsonExample: 99999,
+                    shape: ExampleTypeReferenceShape.primitive(ExamplePrimitive.long(99999))
+                };
+            case "DATE_TIME":
+                return {
+                    jsonExample: "2024-01-01T00:00:00Z",
+                    shape: ExampleTypeReferenceShape.primitive(
+                        ExamplePrimitive.datetime(new Date("2024-01-01T00:00:00Z"))
+                    )
+                };
+            case "UUID":
+                return {
+                    jsonExample: "d5e9c84f-c2b2-4bf4-b4b0-7ffd7a9ffc32",
+                    shape: ExampleTypeReferenceShape.primitive(
+                        ExamplePrimitive.uuid("d5e9c84f-c2b2-4bf4-b4b0-7ffd7a9ffc32")
+                    )
+                };
+            case "DATE":
+                return {
+                    jsonExample: "2024-01-01",
+                    shape: ExampleTypeReferenceShape.primitive(ExamplePrimitive.date("2024-01-01"))
+                };
+            case "BASE_64":
+                // TODO(amckinney): Add support for base64 example primitives; use a string for now.
+                return {
+                    jsonExample: "SGVsbG8gV29ybGQ=",
+                    shape: ExampleTypeReferenceShape.primitive(
+                        ExamplePrimitive.string({ original: "SGVsbG8gV29ybGQ=" })
+                    )
+                };
+            default:
+                assertNever(primitiveType);
+        }
+    }
+
+    private generateExampleUnknown(): ExampleTypeReference {
+        return {
+            jsonExample: {},
+            shape: ExampleTypeReferenceShape.unknown({})
```

Comment:
for `unknown` the OpenAPI example generator creates a map `{ "key": "value" }` unless it is a header, query or path parameter. [code](https://github.com/fern-api/fern/blob/9f1b2844bc4cdebc6e25e0bdcabf6acff94a4d12/packages/cli/openapi-parser/src/schema/examples/ExampleTypeFactory.ts#L241-L249)

Reply:
Updated!